_pages/fsa/install/building-the-fsa-storefront-from-libraries-1-0.md

```diff
@@ -50,15 +50,29 @@ The following procedure describes how to create a new Angular application with t
 
 The easiest way to start a new project is to use Angular Schematics to quickly set up your application. 
 
+In the Windows environment, in order to successfully install the project, first make sure that you have installed GitBash. 
+The next step would be to create the .npmrc file at the root of your Angular project. 
+The .npmrc file should contain the following: 
+
+```shell
+shell = "{instalation directory}\\Git\\bin\\bash.exe" 
+script-shell = "{instalation directory}\\Git\\bin\\bash.exe" 
+
+
+Example:
+shell = "C:\\Program Files\\Git\\bin\\bash.exe"
+script-shell = "C:\\Program Files\\Git\\bin\\bash.exe"
+```
+
```

Comment:
Please add some sentence that after this configuration following installation commands should be executed from GitBash console.